src/types.ts

```diff
@@ -806,6 +806,7 @@ export interface Blocklist extends DbBlocklist {}
 
 export interface DbAllowance {
   createdAt: string;
+  enabledFor?: string[];
```

Comment:
what about `providers`? is there any downside of this name? I feel like it's more specific

Reply:
allowance are set not just by provider. but also during the payments. for eg. while doing a lnurl-pay to the website, in future we might add some other entries there as well, isn't it good to keep it generalized. so that it can hold any kind of entries